### ISSUE TITLE
Commands to implement 

### ISSUE THREAD

**Issue by Meronat** [LanternPowered/LanternServer#38]
This is waiting on 1.13 and the command refactor as we would have to recreate a lot of them then anyways.

**MC Commands To Implement**
---

- [ ] /advancement

- [ ] /blockdata

- [ ] /clear

- [ ] /clone -

- [ ] /debug idk

- [ ] /defaultgamemode

- [ ] /effect

- [ ] /enchant

- [ ] /entitydata

- [ ] /execute

- [ ] /fill

- [ ] /function - Functions are complicated

- [ ] /give

- [ ] /locate - We need to make this better than Minecraft's

- [ ] /recipe 

- [ ] /reload - Can reload more than loot tables, advancements, and functions. Possibly lantern configs

- [ ] /replaceitem

- [ ] /save-all - Is this necessary?

- [ ] /save-on and /save-off - Kind of strange. Low priority

- [ ] /setblock

- [ ] /setmaxplayers - This is a PE only feature, maybe we should add it as a rule? Or is it somewhere already?

- [ ] /spawnpoint

- [ ] /spreadplayers - This command is cool

- [ ] /stats

- [ ] /summon 

- [ ] /tellraw - Not sure how useful this is

- [ ] /testfor These test for commands are strange

- [ ] /testforblock

- [ ] /testforblocks

- [ ] /trigger

- [ ] /wsserver - Okay this is PE only but could be interesting to implement

- [ ] /xp

**Sponge/Other Commands To Implement**
---

- [ ]  /get-data

- [ ] /spawn - Makes sense to spawn the other spawn commands

- [ ] /lantern version

- [ ] /lantern plugin list - Not sure how to set up this and the other plugin commands

- [ ] /lantern plugin info 

- [ ] /lantern plugin reload

- [ ] /lantern plugin reloadall

- [ ] /lantern timings

- [ ] /lantern info/chunk/etc

**Comment by kinggoesgaming** [comment]
The command list should be updated according to the 1.13 refactor
